migrations/20220105170000-add-platformTip-to-orders.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Orders', 'platformTipAmount', { type: Sequelize.INTEGER, defaultValue: 0 });
+    await queryInterface.addColumn('OrderHistories', 'platformTipAmount', { type: Sequelize.INTEGER, defaultValue: 0 });
+    await queryInterface.addColumn('Orders', 'platformTipEligible', { type: Sequelize.BOOLEAN, defaultValue: false });
+    await queryInterface.addColumn('OrderHistories', 'platformTipEligible', {
+      type: Sequelize.BOOLEAN,
+      defaultValue: false,
```

Comment:
I would default to `NULL`, it's problematic to mark it `FALSE` when we actually don't know or if it's unclear.